packages/ts/react-crud/src/autoform-field.tsx

```diff
@@ -132,8 +139,16 @@ export type AutoFormFieldProps = CheckboxProps &
 
 export function AutoFormField(props: AutoFormFieldProps): JSX.Element | null {
   const { form, propertyInfo, options } = props;
-  const label = options?.label ?? propertyInfo.humanReadableName;
-  if (options?.renderer) {
+  const label = options.label ?? propertyInfo.humanReadableName;
+
+  const formPart = useFormPart(getPropertyModel(form, propertyInfo));
+  const defaultValidators = useMemo(() => formPart.validators, []);
+  const { validators } = options;
+  useEffect(() => {
+    formPart.setValidators([...defaultValidators, ...(validators ?? [])]);
+  }, [validators]);
```

Comment:
It seems that this code doesn't have any side effects. 
For now, I think there are no other simple solutions for users to add custom validators. 

An already-ready option is for users to use a custom layout where the form object is provided to set any validators they want. 
Other options are to provide some callbacks that interact with formPart/binder directly for the model property.

Reply:
@sissbruecker  was concerned that setValidators would cause a change event in Hilla Binder instance, which in turn updates some state in useForm.

Reply:
I have tried it, and it seems to not have any circular recursion.